test/addons/dlopen-ping-pong/binding.cc

```diff
@@ -0,0 +1,48 @@
+#include <node.h>
+#include <v8.h>
+
+#ifndef _WIN32
+
+#include <dlfcn.h>
+
+extern "C" const char* dlopen_pong(void) {
+  return "pong";
+}
+
+namespace {
+
+using v8::FunctionCallbackInfo;
+using v8::Isolate;
+using v8::Local;
+using v8::Object;
+using v8::String;
+using v8::Value;
+
+typedef const char* (*ping_t)(void);
```

Comment:
`_t` suffix is technically reserved.

Reply:
It's just a test. Does it matter?

Reply:
It doesn't, that's why I said "technically" 😛

Anyways it's up to you if you want to change it.

Reply:
@TimothyGu Just curious, wouldn’t that just be for C anyway? C++ has namespacing and this is even inside an anonymous one…